src/main/resources/application.properties

```diff
@@ -1 +1,19 @@
+spring.jpa.hibernate.ddl-auto=create
+spring.datasource.url=jdbc:mysql://localhost:3306/fundraiserdb
+spring.datasource.username=root
+spring.datasource.password=
 
+spring.profiles.active=dev
+
+spring.servlet.multipart.max-file-size=1024KB
+spring.servlet.multipart.max-request-size=1024KB
+
+spring.kafka.consumer.bootstrap-servers=localhost:9092
+spring.kafka.consumer.group-id=enterprise
+spring.kafka.consumer.auto-offset-reset=earliest
+spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.consumer.value-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+
+spring.kafka.producer.bootstrap-servers=localhost:9092
+spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
+spring.kafka.producer.value-serializer=org.apache.kafka.common.serialization.StringSerializer
```

Comment:
In the application.properties file, I inserted code that allows you to connect to the database. Spring Boot provides defaults for everything. You must define the connection attributes in the application.properties file if you wish to use any database.
 
This is a helpful link: https://www.youtube.com/watch?v=j8KUiRPGSPs&list=PL73qvSDlAVVhr2V85gxReq9PUptNaVI9M&index=44&ab_channel=BrandanJones